pkg/runtime/optimize/dml/select.go

```diff
@@ -94,9 +98,27 @@ func optimizeSelect(ctx context.Context, o *optimize.Optimizer) (proto.Plan, err
 				}
 				valueList = append(valueList, calculateRes)
 				columnList = append(columnList, stmt.Select[i].DisplayName())
-
+			case *ast.SelectElementColumn:
```

Comment:
Should we detect if there's from clause followed?

Reply:
As stated in the issue, no FROM followed.

Reply:
Yeah, I know it, but if I write this sql with from followed which is a wrong SQL, should I report an error?